### PR TITLE
Value typo in has_gte_10k_favs

### PR DESCRIPTION
**Pull request by AndroidDev77**
has_gte_10k_favs has typo in threshold
1,000 ->10,000

### CODE REVIEW

**Comment by kenluzuriaga0** [comment]
you saved twitter

File: home-mixer/server/src/main/scala/com/twitter/home_mixer/functional_component/decorator/HomeTweetTypePredicates.scala [hunk 1]
```diff
@@ -160,7 +160,7 @@ object HomeTweetTypePredicates {
     ("has_gte_1k_favs", _.getOrElse(EarlybirdFeature, None).exists(_.favCountV2.exists(_ >= 1000))),
     (
       "has_gte_10k_favs",
-      _.getOrElse(EarlybirdFeature, None).exists(_.favCountV2.exists(_ >= 1000))),
+      _.getOrElse(EarlybirdFeature, None).exists(_.favCountV2.exists(_ >= 10000))),
```
Comment (kapilepatel) [inline home-mixer/server/src/main/scala/com/twitter/home_mixer/functional_component/decorator/HomeTweetTypePredicates.scala:163]: Looks good, comparing to the other values it is should be 10 k.

**Review by kapilepatel — COMMENTED** [review]
Looks good to me.

**Review by Radu-D — COMMENTED** [review]
👍

**Comment by Merrick-Clay** [comment]
Good find!

**Comment by AndroidDev77** [comment]
This can be merged and closed now

**Pull request closed without merging**